test/cpp/interop/interop_client.cc

```diff
@@ -1021,6 +1021,7 @@ bool InteropClient::DoOrcaPerRpc() {
 }
 
 bool InteropClient::DoOrcaOob() {
+  static constexpr auto TIMEOUT = absl::Seconds(5);
```

Comment:
This should be named `kTimeout`, as per https://google.github.io/styleguide/cppguide.html#Constant_Names.

Reply:
Done